pkgs/build-support/rust/lib/default.nix

```
@@ -30,20 +30,27 @@ rec {
     else lib.optional platform.isUnix "unix"
       ++ lib.optional platform.isWindows "windows";
 
+  # https://doc.rust-lang.org/reference/conditional-compilation.html#target_vendor
+  toTargetVendor = platform: let
+      inherit (platform.parsed) vendor;
+      vendor_ = platform.rustc.platform.vendor or {
+        "w64" = "pc";
+      }.${vendor.name} or vendor.name;
+    in
+      vendor_;
```

lgtm, just a formatting suggestion because I don't like the 4 line indentation and the name `vendor_` that we can avoid
```suggestion
  toTargetVendor = platform:
    let
      inherit (platform.parsed) vendor;
    in
    platform.rustc.platform.vendor or {
      "w64" = "pc";
    }.${vendor.name} or vendor.name;
```

---

I can just run nixpkgs-fmt in a first commit. I was trying to keep it consistent with the rest of the file.

---

You don't have to run nixpkgs-fmt. I'm not exactly sure what you are referring to in terms of consistency because I don't see any other line with a 4-space indentation, but this might be more consistent with the formatting of the rest of the file
```suggestion
  toTargetVendor = platform: let
    inherit (platform.parsed) vendor;
  in platform.rustc.platform.vendor or {
    "w64" = "pc";
  }.${vendor.name} or vendor.name;
```

---

Ah sorry about that, missed your point about the two extra spaces. All fixed.